tronweb/coffee/views.coffee

```diff
@@ -34,9 +34,14 @@ window.dateFromNow = (string, defaultString='never') ->
 
 
 window.getDuration = (time) ->
+    days = '0'
+    if time.indexOf("day") != -1 or time.indexOf("days") != -1
```

Comment:
Should `if time.indexOf("day") != -1` be enough here?

Reply:
Oh... yes.